environment_arm.yml

```diff
@@ -9,12 +9,12 @@ dependencies:
   - boost >=1.76.0,<1.82
   - boost-cpp >=1.76.0,<1.82
   - blas
-  - nasa-ames-stereo-pipeline::bullet
+  - usgs-astrogeology::bullet
   - bz2file
   - bzip2
   - cmake >=3.27,<3.28
   - cspice =67
-  - nasa-ames-stereo-pipeline::csm >=3.0.3,<3.0.4
+  - usgs-astrogeology::csm >=3.0.3,<3.0.4
```

Comment:
Remove `usgs-astrogeology::`. CSM has been updated on conda-forge.

Reply:
Yes, indeed, has been updated by Kelvin. I removed that in latest push. Thank you for reviewing.